mix.exs

```diff
@@ -24,7 +24,7 @@ defmodule Avalon.MixProject do
       {:ex_doc, "~> 0.36", only: :dev, runtime: false},
       {:jason, "~> 1.0"},
       {:nimble_options, "~> 1.0"},
-      {:req, "~> 0.5", optional: true, only: :test},
+      {:req, "~> 0.5"},
```

Comment:
No, this must be optional. The clients are meant for drop-in ease and are entirely optional to using the library. I don't want to make this assumption for library users.